plugins/CoreHome/angularjs/widget-loader/widgetloader.directive.js

```diff
@@ -35,6 +37,9 @@
 
                 return function (scope, element, attrs, ngModel) {
                     scope.widgetName = attrs.widgetName;
+                    scope.errorFaqLink = isGeneralSettingsAdminEnabled || isPluginsAdminEnabled ?
+                        _pk_translate('General_ErrorRequestFaqLink', ['<a target="_blank" href="https://matomo.org/faq/troubleshooting/faq_19489/">', '</a>'])
```

Comment:
fyi this needs a rel="noreferrer noopener" attribute for privacy and security

Reply:
OK, I added it. Good to know about this attribute and these values! I did not know these values and what they mean.